ckb-bin/src/subcommand/run.rs

```diff
@@ -74,6 +81,7 @@ pub fn run(args: RunArgs, version: Version) -> Result<(), ExitCode> {
             .cloned()
             .unwrap_or_default(),
         args.config.tmp_dir.as_ref(),
+        args.assume_valid_target,
```

Comment:
I tend not to add this configuration in the config, It is not a parameter that affects behavior at all times. It only needs to be paid attention to when the node is initialized and synchronized. 

Of course, it can be said that this reason is not very good, but I personally think so

Reply:
You may use `#[serde(skip)]` to allow the user to set this value only with a cli argument, but it still makes sense to put this field on `SyncConfig` struct. My personal preference is to be able to support config file as well, let's leave it open to discussion.